BraveWallet/Panels/WalletPanelView.swift

```diff
@@ -188,18 +200,32 @@ struct WalletPanelView: View {
   var body: some View {
     ScrollView(.vertical, showsIndicators: false) {
       VStack(spacing: 0) {
-        Text("Brave Wallet")
-          .font(.headline)
-          .padding(16)
-          .frame(maxWidth: .infinity)
-          .overlay(
-            Color.white.opacity(0.3) // Divider
-              .frame(height: pixelLength),
-            alignment: .bottom
-          )
-          .background(
-            Color.clear
-          )
+        HStack {
+          Button {
+            presentWalletWithContext(.default)
+          } label: {
+            Image(systemName: "arrow.up.left.and.arrow.down.right")
+              .rotationEffect(.init(degrees: 90))
+          }
+          Spacer()
+          Text(Strings.Wallet.braveWallet)
+            .font(.headline)
+            .background(
+              Color.clear
+            )
+          Spacer()
+          Button {
+            presentWalletWithContext(.settings)
```

Comment:
Can we make this a context menu and add lock wallet option to match main wallet?